### PR TITLE
chore: Member UI 정리

### PR DESCRIPTION
**Pull request by myeongjunkim**
디자인 시안에 맞춰 간격과 색깔, 그림자 등을 조절하였습니다.
컴포넌트 단위로 작업계속 이어갈 예정입니다.

UI 작업이기 때문에 Preview 로 확인 부탁드립니다!

[Visit Preview](https://vercel.live/open-feedback/prayu-staging-git-fix-ui-detail-visioneers-projects.vercel.app?via=pr-comment-visit-preview-link&passThrough=1)

### CODE REVIEW

**Comment by myeongjunkim** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/TeamVisioneer/PrayU-Web/82?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#82** <a href="https://app.graphite.dev/github/pr/TeamVisioneer/PrayU-Web/82?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#79** <a href="https://app.graphite.dev/github/pr/TeamVisioneer/PrayU-Web/79?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @myeongjunkim and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/components/member/MyMember.tsx [hunk 1]
```diff
@@ -67,38 +67,39 @@ const MyMember: React.FC<MemberProps> = ({ currentUserId, groupId }) => {
...
+                  }
+                </p>
               </div>
             );
           })}
         </div>
-        <p className=" flex items-center text-gray-500 text-[8px] p-2">
-          누군가 내게 기도했어요 😊
+        <p className=" flex items-center text-gray-500 text-[10px]">
+          내게 기도한 사람이 있어요 😊
```
Comment (myeongjunkim) [inline src/components/member/MyMember.tsx:102]: @yaminsu5674 
이 부분 워딩 수정해보았는데 확인 부탁드립니다!

**Review by treephesians — APPROVED** [review]
LGTM

File: src/components/member/MyMember.tsx (hunk 1 above)
Comment (yaminsu5674) [reply src/components/member/MyMember.tsx:102]: 뭔가 더 토스스럽네요 ㅋㅋㅋㅋ 굿굿입니다!

**Review by yaminsu5674 — APPROVED** [review]
고생하셨습니다:)

**Pull request merged**